src/test/java/com/google/jenkins/plugins/computeengine/integration/ITUtil.java

```diff
@@ -160,6 +136,50 @@ static String format(String s) {
         return String.format(s, PROJECT_ID);
     }
 
+    private static String bootDiskImageName() {
+        String projectId = System.getenv("GOOGLE_BOOT_DISK_PROJECT_ID");
+        if (projectId == null) {
+            projectId = PROJECT_ID;
+        }
+
+        String imageName = System.getenv("GOOGLE_BOOT_DISK_IMAGE_NAME");
+        if (imageName == null) {
+            imageName = "jenkins-gce-integration-test-jre"; /* Image built with packer, see the `setup-gce-image.sh` */
+        }
+
+        String imageFqn = String.format("projects/%s/global/images/%s", projectId, imageName);
+        log.info("Using boot disk image: " + imageFqn);
+
+        return imageFqn;
+    }
+
+    private static String bootDiskProject() {
+        String project = System.getenv("GOOGLE_BOOT_DISK_PROJECT_ID");
+        project = project != null ? project : PROJECT_ID;
+        log.info("Using boot disk project: " + project);
+        return project;
+    }
+
+    private static String startUpScript() {
```

Comment:
needs to do some work on the windows tests.
should be moved to packer same as ways as linux